test/fixtures/v5/rotary/default_exit.json

```diff
@@ -0,0 +1,22 @@
+{
+    "step": {
+        "maneuver": {
+            "modifier": "left",
+            "type": "rotary"
+        },
+        "name": "Way Name",
+        "exits": "4A,4B"
+    },
+    "instructions": {
+        "de": "In den Kreisverkehr fahren und auf Way Name verlassen",
+        "en": "Enter the rotary and exit onto Way Name",
```

Comment:
In complex roundabouts/rotaries/traffic circles it could make sense to add the exit information if we have it.